compiler/damlc/tests/daml-test-files/ContractKeyNotVisible.daml

```diff
@@ -118,11 +116,4 @@ localFetch = scenario do
   cid <- submit p1 $ create (LocalKeyVisibility p1 p2)
   submit p2 $ exercise cid LocalFetch
 
-localLookupFetchMustFail = scenario do
```

Comment:
Removed as:
- lookup and fetch are tested above
- lookup followed by fetch will fail due to contract being consumed